content/en-us/luau/functions.md

```diff
@@ -115,15 +119,31 @@ testButton:changeEnabled(false) -- false
 
 ## Callbacks
 
-Callbacks are functions that execute in response to another function or process. Some `Global.RobloxGlobals` functions, such as `delay()` and `spawn()`, take callbacks as parameters. In the Roblox Studio API, callbacks are write-only members of some classes. Unlike [event handlers](#event-handlers), callbacks yield until they return. Widely used callbacks include:
+Callbacks are functions that execute in response to another function or process.
+
+### Basic Callbacks
+
+Functions can be passed into other functions, for example, an [anonymous](#anonymous-functions) function can be used to implement a callback that `Library.table.sort()` then uses to sort a list of `Class.Player`|players from `Class.Players.GetPlayers()`.
+
+```lua
+local Players = game:GetService("Players")
+local sortedPlayers = Players:GetPlayers()
+
+table.sort(sortedPlayers, function(a, b)
+	-- use an anonymous callback to sort players by name
+	return a.Name < b.Name
+end)
+```
+
+### Roblox Callbacks
```

Comment:
```suggestion
Functions can be passed into other functions, for example, an [anonymous](#anonymous-functions) function can be used to implement a callback that `Library.table.sort()` then uses to sort a list of `Class.Player|Players` from `Class.Players.GetPlayers()`.

```lua
local Players = game:GetService("Players")
local sortedPlayers = Players:GetPlayers()

table.sort(sortedPlayers, function(a, b)
	-- Use an anonymous callback to sort players by name
	return a.Name < b.Name
end)
```

### Roblox Callbacks
```